src/NJsonSchema.CodeGeneration.TypeScript/Templates/Interface.liquid

```diff
@@ -12,4 +12,60 @@
 
     [key: string]: {{ IndexerPropertyValueType }}; 
 {% endif -%}
+}
+
+{% if ExportTypes %}export {% endif %}class {{ ClassName }}ParsingUtils {
```

Comment:
Only generate when setting is true

Reply:
This is basically a copy of the class logic, right? Cant we move it to a new template and reuse it here and in class?

Reply:
Rename to {{ ClassName }}Serializer?

Reply:
Rename to {{ ClassName }}Serializer?